src/main/java/com/hubspot/jackson/jaxrs/PropertyFilteringMessageBodyWriter.java

```diff
@@ -75,9 +79,18 @@ public void writeTo(Object o, Class<?> type, Type genericType, Annotation[] anno
     }
   }
 
-  private Collection<String> getProperties(String name) {
-    List<String> values = uriInfo.getQueryParameters().get(name);
 
+  private Collection<String> getQueryProperties(String name) {
+    return getProperties(uriInfo.getQueryParameters().get(name));
+  }
+
+
+  private Collection<String> getDefaultProperties(String[] values) {
+    return getProperties(Arrays.asList(values));
```

Comment:
This only calls `getProperties` to allow comma-separated right? Seems unnecessary for values specified in the annotation